github/functions.sh

```diff
@@ -0,0 +1,57 @@
+set +x
+# set -e
+# TODO(kwk): Is there a better way to check project existence?
+# TODO(kwk): Maybe: copr list $username | grep --regexp="^Name: \$project$"
+function project_exists(){
+  local project=$1;
+  copr get-chroot $project/fedora-rawhide-x86_64 > /dev/null 2>&1 \
+  && echo "true" || echo "false";
+}
+
+function get_active_build_ids(){
+  local project=$1;
+  copr list-builds --output-format text-row $project \
+    | grep --perl-regexp --regexp='(running|waiting|pending|importing|starting)' \
+    | cut -f 1
+}
+
+# Prints the chroots we care about.
+function get_chroots() {
+  copr list-chroots | grep -P '^fedora-(rawhide|[0-9]+)' | tr '\n' ' '
+}
+
+# Prints the packages we care about
+function get_packages() {
+  echo "python-lit llvm clang lld compiler-rt libomp"
+}
+
+# Returns false if a package needs special handling on certain architectures
+function is_package_supported_by_chroot() {
+  local pkg=$1;
+  local chroot=$2;
+
+  if [[ ("$pkg" == "lld" || "$pkg" == "libomp") && $chroot =~ -s390x$ ]]; then
+    false
+  else
+    true
+  fi
```

Comment:
I think this can be reduced to just this?
```suggestion
  ! [[ ("$pkg" == "lld" || "$pkg" == "libomp") && $chroot =~ -s390x$ ]]
```

Reply:
While it can be shortened, I dislike it.